.github/workflows/test.yml

```diff
@@ -6,8 +6,17 @@ jobs:
   test-react:
     name: Test React
     runs-on: ubuntu-latest
+    container:
+      image: cypress/browsers:node14.17.0-chrome91-ff89
+      options: --privileged
```

Comment:
running in privileged mode to allow the max number of watchers to be increased--otherwise docker mounts sysctl.conf as readonly. This could be excluded...we'd just get a bunch of errors about running out of watchers like we've had in the past.